assets/queries/cloudFormation/logging_is_disabled/test/positive.yaml

```diff
@@ -0,0 +1,33 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Resources:
+  myDistribution1:
+    Type: AWS::CloudFront::Distribution
+    Properties:
+      DistributionConfig:
```

Comment:
To make this sample valid. Add mandatory **DefaultCacheBehaviour** section inside **DistributionConfig** e.g:
```       
DefaultCacheBehavior:
    AllowedMethods:
    - GET
    - HEAD
    - OPTIONS
    TargetOriginId: myS3Origin
    ForwardedValues:
      QueryString: 'false'
      Cookies:
        Forward: none
    TrustedSigners:
    - 1234567890EX
    ViewerProtocolPolicy: allow-all
```